umich_sim/wizard/inputs/g920_wheel.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 from .base_wheel import BaseWheel
 from . import ClientMode, WheelKeyType, ControlEventType
+from evdev import ecodes, ff
```

Comment:
global imiport might break windows version? (not sure)

Reply:
put it into TODO